apps/docs/src/app/core/api-files.ts

```diff
@@ -72,6 +72,9 @@ export const API_FILES = {
         'DynamicPageFooterComponent',
         'DynamicPageContentComponent'
     ],
+    facets: [
+      'FacetComponent'
```

Comment:
Could you add `'FacetGroupComponent'` there as well?

Reply:
thanks for noticing, added this too.